src/control/common/proto/mgmt/mgmt_grpc.pb.go

```diff
@@ -1,6 +1,12 @@
+//
+// (C) Copyright 2019-2022 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
```

Comment:
I'm not sure that these generated files should have copyright headers. Did you add them manually?

Reply:
not intentionally, I presume it was the precommit hook, I will remove them, done